examples/quill.html

```diff
@@ -36,14 +36,23 @@
       textLogHolder.innerText = doc.getRoot().content.getAnnotatedString();
     }
 
+    function embedAttributeToContent(val) {
+      const { embed, ...attributes } = val.attributes ?? {};
+
+      const delta = embed
+        ? { insert: JSON.parse(embed), attributes } :
+        {
+          insert: val.content || ' ',
+          attributes: val.attributes,
+        };
+      return delta;
+    }
+
```

Comment:
`change.content` may be missing if delivered through `deltaOp`, so it would be better to receive the change directly.

```suggestion
    function toDelta(change) {
      const { embed, ...attributes } = change.attributes ?? {};

      const delta = embed
        ? { insert: JSON.parse(embed), attributes } :
        {
          insert: change.content || '',
          attributes: change.attributes,
        };
      return change;
    }

```

We also need to rename the previous `toDelta` function to `toDeltaList`.